<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotateAlign</class>
 <widget class="QMainWindow" name="RotateAlign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="grid_layout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontal_layout">
        <item>
         <layout class="QGridLayout" name="grid_layout_x">
          <item row="1" column="0">
           <widget class="QSlider" name="slider_distance_x">
            <property name="minimum">
             <number>-6400</number>
            </property>
            <property name="maximum">
             <number>6400</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_distance_x">
            <property name="text">
             <string>Distance x</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="number_distance_x">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="grid_layout_y">
          <item row="0" column="0">
           <widget class="QLabel" name="label_distance_y">
            <property name="text">
             <string>Distance Y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="number_distance_y">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="slider_distance_y">
            <property name="maximum">
             <number>4800</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="grid_layout_z">
          <item row="1" column="1">
           <widget class="QLCDNumber" name="number_distance_z">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="slider_distance_z">
            <property name="maximum">
             <number>20480</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_distance_z">
            <property name="text">
             <string>Distance Z</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_button">
          <property name="text">
           <string>Rotate and Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QVTKWidget" name="pcl_widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
